flytekit/common/constants.py

```diff
@@ -38,3 +38,13 @@ class CloudProvider(object):
     AWS = "aws"
     GCP = "gcp"
     LOCAL = "local"
+
+
+class StructuredDatasetProtocol(object):
```

Comment:
we should delete these two classes.  The point of making the format/protocols just plain strings is that they're easily extensible.

We can have parquet be the default format, but we should just have a string constant in the structured_dataset.py file.  This file (and everything else under `flytekit/common`) is getting deleted in january.

Reply:
got it, will move to structured_dataset.py